sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -621,7 +624,7 @@ def upload_file(
 
     @distributed_trace
     def start_copy_from_url(self, source_url, **kwargs):
-        # type: (str, Any) -> Any
+        # type: (str, Any) -> dict[str, Any]
```

Comment:
I think this one needs to stay uppercase `Dict[str, Any]` as its a real type hint.

Reply:
Sounds good. I'll keep to this convention. I also double-checked on the docs that `dict` is linking to the right place in Python, but I think the real type hint does require `Dict` instead of `dict`.